exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java

```diff
@@ -126,8 +127,12 @@ CloseableRecordBatch getReaderBatch(FragmentContext context, EasySubScan scan) t
     final ImplicitColumnExplorer columnExplorer = new ImplicitColumnExplorer(context, scan.getColumns());
 
     if (!columnExplorer.isSelectAllColumns()) {
+      // We must make sure to pass a table column (not to be confused with implicit column) to the underlying record reader.
+      List<SchemaPath> tableColumns =
```

Comment:
Ok, I see.   Although this patch resolves this issue, I am thinking that without doing a performance test it is not feasible to see the performance impact of the overall implicit columns support.  It is a nice feature to have but I think we can give it a little more time to go through functional and perf tests.